### DISCUSSION TITLE
Debug messages Medicine

### DISCUSSION THREAD

**Discussion by lifof** [cagnulein/qdomyos-zwift#345] [Show and tell]
Finally found it :smile: 

~~add this piece of code after the includes in bluetooth.h and remove the debug config from the project file add optimize_full and ltcg and build in release mode.~~

use the Fifth wave of code chages commit instead

```


// DONE: Eureka found a solution for the debug messages only problem is how make them sustain this state
// NOTE: Needs testing
#undef QT_MESSAGELOG_FILE
#undef QT_MESSAGELOG_LINE
#undef QT_MESSAGELOG_FUNC

#define QT_MESSAGELOG_FILE static_cast<const char *>(__FILE__)
#define QT_MESSAGELOG_LINE __LINE__
#define QT_MESSAGELOG_FUNC static_cast<const char *>(Q_FUNC_INFO)
```

Tell me if you're satisfied with the results to include this in the project.

Have great day :+1:

**Comment by cagnulein** [comment]
Yes i Guess it perfect!

Il sab 5 giu 2021, 18:18 lifof ***@***.***> ha scritto:
